treeStructure.py

```diff
@@ -94,11 +94,12 @@ def write(self):
         return
 
 class gemDacCalTreeStructure(gemGenericTree):
-    def __init__(self, name, nameX, nameY, isGblDac=True, storeRoot=False, description="Generic GEM DAC Calibration Tree"):
+    def __init__(self, name, dacSelect, nameX, nameY, isGblDac=True, storeRoot=False, description="Generic GEM DAC Calibration Tree"):
```

Comment:
Don't have this be a required entry, make it default to `-1`.  As it is this change would break `trimChamberV3.py`:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/16d6ae2bbd5d72f7c1e69ea38fae0b744d8bb4cf/trimChamberV3.py#L222-L243

In this context `dacSelect` is not needed (since it has no bearing there).

Reply:
done

Reply:
Doesn't look like this has been pushed to your remote.

Reply:
@bdorney, yes, sorry treeStructure.py was not added in the commit

it is fixed now